extensions/amp-analytics/0.1/crypto-impl.js

```diff
@@ -70,9 +83,7 @@ export class Crypto {
    * @throws {!Error} when input string contains chars out of range [0,255]
    */
   sha384Base64(input) {
-    return this.sha384(input).then(buffer => {
-      return lib.base64(buffer);
```

Comment:
Ohh, we might have to update https://github.com/ampproject/amphtml/blob/fa5c6cc7a9a78802bc816b6a986543420d90917f/third_party/closure-library/compile.sh, too.